### PR TITLE
Add events to CommitOrderProcessor cleanUpPendingOrders method

### PR DESCRIPTION
**Pull request by CiLLiTxBanger**
It would be super cool if the cleanUpPendingOrders method would throw events before setting a pending order to "aborted".
In my specific UseCase i want to do some other stuff aswell, if the order gets cleaned up by e.g. the maintenance cron job.

I tested these Events and they worked well for me.
I hope this gets pulled :)

### CODE REVIEW

**Comment by CiLLiTxBanger** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/CheckoutManager/V7/CommitOrderProcessor.php [hunk 1]
```diff
@@ -329,7 +330,6 @@ public function cleanUpPendingOrders(): void
 
         //Abort orders with payment pending
         $list = $orderManager->buildOrderList();
-        $list->addFieldCollection('PaymentInfo');
```
Comment (Corepex) [inline src/CheckoutManager/V7/CommitOrderProcessor.php:332]: Why was this line removed?

**Review by Corepex — COMMENTED** [review]
@CiLLiTxBanger, thanks for your PR. :1st_place_medal:

**Comment by Corepex** [comment]
@CiLLiTxBanger friendly reminder :alarm_clock:

File: src/CheckoutManager/V7/CommitOrderProcessor.php (hunk 1 above)
Comment (CiLLiTxBanger) [reply src/CheckoutManager/V7/CommitOrderProcessor.php:332]: i removed this line because it serves no purpose.
The fieldCollection "PaymentInfo" is not used here and gets added in line 354 anyways (when it's needed).

**Review by Corepex — APPROVED** [review]
LGTM - thanks for the PR. :100:  :1st_place_medal:
